### DISCUSSION TITLE
Different font sizes for panes

### DISCUSSION THREAD

**Discussion by octavz** [wezterm/wezterm#703] [Ideas]
Hi,

Not sure if this is already possible, I looked and couldn't find it by myself.

In Windows Terminal, I can set for splits (panes), different font sizes which is very useful when the development setup is composed from 2 terminals (ie editor + compiler), in compiler split I want smaller font so I can fit longer lines (errors).

If is not already there and possible would be a nice addition even under a setting.

Thanks

**Comment by wez** [comment]
It's not possible today, as the entire tab contents render within the same grid of cells as a performance optimization.  I think it would be nice to support this, but it's not a simple change.

**Reply by rofrol** [reply]
Ghostty supports this.

**Comment by futile** [comment]
Hi, just was wondering about the same question, and found this existing thread :) Has anything changed about this since then?

Otherwise I'll use a workaround or something. My use case is that I would like nvim to use a different font size (maybe font?) than the rest of my terminal, without changing the appearance of all panes, tabs, etc. Any other ideas to solve this also welcome :)

**Reply by wez** [reply]
No changes to this since that last comment

**Reply by metalelf0** [reply]
No workaround AFAIK, but different windows + a tiling window manager kinda achieves that. Suboptimal as additional navigation keys are required, but for simple scenarios it's viable.

**Comment by rofrol** [comment]
Ghostty can do this.
